test/e2e/antreapolicy_test.go

```diff
@@ -1944,7 +1944,10 @@ func testANPMultipleAppliedTo(t *testing.T, singleRule bool) {
 	}
 
 	t.Logf("Making the Policy apply to y/c by labeling it with the temporary label that matches the dummy AppliedTo")
-	podYC, _ := k8sUtils.GetPodByLabel("y", "c")
+	podYC, err := k8sUtils.GetPodByLabel("y", "c")
+	if err != nil {
+		t.Errorf("Failed to get Pod in Namespace y with label 'pod=c':%v", err)
```

Comment:
Sorry for being nit-picking, could you add space after ":" to keep same log style?